asciidoctor-gradle-jvm/src/gradleTest/complex-jvm-setup/build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+import org.asciidoctor.gradle.jvm.AsciidoctorPdfTask
+import org.asciidoctor.gradle.jvm.AsciidoctorTask
+import org.asciidoctor.gradle.jvm.OutputOptions
+import org.asciidoctor.gradle.jvm.ProcessMode
+
+plugins {
+    id("org.asciidoctor.jvm.convert")  version "2.0.0-alpha.2"
```

Comment:
Adding the version is unnecessary as the plugin get atoumatically detected by the classpath as set by GradleTestKit.